app/controllers/api/feedback_submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  class FeedbackSubmissionsController < ApiController
+    def create
+      @feedback = FeedbackSubmission.new(feedback_params)
+
+      if @feedback.save
+        ZendeskTicketsJob.perform_later(@feedback)
+      else
+        render status: 422, json: { messages: @feedback.errors.full_messages }
```

Comment:
I haven't used `ActiveModel::Serializer` – in principle it looks good, but since this API is currently extremely simple I don't think it's necessary to add right now.